atlasdb-client/src/main/java/com/palantir/atlasdb/schema/CleanupSchema.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.schema;
+
+import java.io.File;
+
+import com.palantir.atlasdb.keyvalue.api.Namespace;
+import com.palantir.atlasdb.schema.queue.SequentialTables;
+import com.palantir.atlasdb.table.description.OptionalType;
+import com.palantir.atlasdb.table.description.Schema;
+
+public enum CleanupSchema implements AtlasSchema {
```

Comment:
can we name this a bit more descriptively?

Reply:
Yep, I'll update this to `CleanupTaskQueueSchema`, or something along those lines if I can think of something better.